### PR TITLE
Update terms wallet modal.

### PR DESCRIPTION
**Pull request by biocom**
Addresses #916 

Screen Shot 2021-07-21 at 17 14 33Screen Shot 2021-07-21 at 17 14 25

### CODE REVIEW

File: src/custom/components/WalletModal/index.tsx [hunk 1]
```diff
@@ -10,6 +12,22 @@ export const GpModal = styled(Modal)`
...
+const TermsWrapper = styled.div`
+  color: ${({ theme }) => theme.text1};
+`
+
+export function CustomTerms() {
+  return (
+    <TermsWrapper>
+      <Trans>
+        By connecting a wallet, you agree to GnosisDAO&apos;s{' '}
+        <ExternalLink href="#/terms-and-conditions">Terms &amp; Conditions</ExternalLink> and acknowledge that you have
```
Comment (W3stside) [inline src/custom/components/WalletModal/index.tsx:24]:
```suggestion
        <ExternalLink href="#/terms-and-conditions">Terms &amp; Conditions</ExternalLink>, and acknowledge that you have
```

File: src/custom/components/WalletModal/index.tsx [hunk 2]
```diff
@@ -10,6 +12,22 @@ export const GpModal = styled(Modal)`
...
+        read, understood and agree to them.{' '}
```
Comment (W3stside) [inline src/custom/components/WalletModal/index.tsx:25]:
```suggestion
        read, understood, and agree to them.{' '}
```

**Review by W3stside — APPROVED** [review]
great job with the mod, just minor suggestions on text. oxford comma

File: src/custom/components/WalletModal/index.tsx [hunk 3]
```diff
@@ -10,6 +12,22 @@ export const GpModal = styled(Modal)`
...
+const TermsWrapper = styled.div`
+  color: ${({ theme }) => theme.text1};
+`
+
+export function CustomTerms() {
+  return (
+    <TermsWrapper>
+      <Trans>
+        By connecting a wallet, you agree to GnosisDAO&apos;s{' '}
+        <ExternalLink href="#/terms-and-conditions">Terms &amp; Conditions</ExternalLink>, and acknowledge that you have
```
Comment (W3stside) [inline src/custom/components/WalletModal/index.tsx:24]:
```suggestion
        <ExternalLink href="#/terms-and-conditions">Terms &amp; Conditions</ExternalLink> and acknowledge that you have
```

**Comment by biocom** [comment]
> great job with the mod, just minor suggestions on text. oxford comma

/ Silke has entered the chat

**Review by alfetopito — APPROVED** [review]

**Comment by W3stside** [comment]
@biocom my only suggestion is to pass the new comp as props as I did in https://github.com/gnosis/cowswap/pull/1002/commits/23b42d91aa01706d69481d5654124455c22afcc0

**Comment by W3stside** [comment]
merging @anxolin please post merge review (coming back from vacay will suuuuck 😅 )

**Comment by biocom** [comment]
@W3stside Sure, will address that post merge.

**Pull request merged**